assistant_dists/dream_kg/pipeline_conf.json

```diff
@@ -385,6 +385,24 @@
                     "component": "components/PbLNvh4hrvs47rPaf2bfYQ.yml",
                     "service": "annotators/combined_classification/service_configs/combined-classification"
                 }
+            },
+            "user_knowledge_memorizer": {
+                "connector": {
+                    "protocol": "http",
+                    "url": "http://user-knowledge-memorizer:8027/respond"
```

Comment:
timeout

Reply:
@Ramimashkouk WHY DID YOU RESOLVE IT?
ADD TIMEOUT key to connector dictionary.it is necessary